src/apps/backend/modules/account/account-service.ts

```diff
@@ -30,8 +31,7 @@ export default class AccountService {
   public static async getOrCreateAccountByPhoneNumber(
     phoneNumber: PhoneNumber
   ): Promise<Account> {
-    let account =
-      await AccountReader.getAccountByPhoneNumberOptional(phoneNumber);
+    let account = await AccountReader.getAccountByPhoneNumber(phoneNumber);
```

Comment:
Why is there a change in formatting? We all should be using checked in formatting and linting rules.